core-bundle/src/Controller/ContentElement/AbstractContentElementController.php

```diff
@@ -34,4 +34,29 @@ public function __invoke(Request $request, ContentModel $model, string $section,
     }
 
     abstract protected function getResponse(Template $template, ContentModel $model, Request $request): Response;
+
+    /**
+     * Set the shared max age considering the start and stop limit of the content element.
+     */
+    protected function responseWithSharedMaxAgeFromStartAndStop(Response $response, ContentModel $model): Response
+    {
+        $time = time();
+        $min = [];
+
+        if ($model->start > $time) {
```

Comment:
To be fully correct, we should check `$model->start !== '' && $model->start > $time`, because right now it only works because `$time` is always greater than 0.